x/queries/grpc_query_test.go

```diff
@@ -25,76 +26,79 @@ func TestCirculating(t *testing.T) {
 		acc2 sdk.AccAddress = rand.Bytes(sdk.AddrLen)
 		acc3 sdk.AccAddress = rand.Bytes(sdk.AddrLen)
 	)
+
 	bkMock := bankKeeperMock{
 		balances: map[string]sdk.Coins{
-			acc1.String(): sdk.NewCoins(sdk.NewCoin("alx", sdk.OneInt())),
-			acc2.String(): sdk.NewCoins(sdk.NewCoin("alx", sdk.NewInt(2))),
-			acc3.String(): sdk.NewCoins(sdk.NewCoin("blx", sdk.NewInt(4))),
-			authtypes.NewEmptyModuleAccount(stakingtypes.BondedPoolName).Address:    sdk.NewCoins(sdk.NewCoin("alx", sdk.NewInt(100))),
-			authtypes.NewEmptyModuleAccount(stakingtypes.NotBondedPoolName).Address: sdk.NewCoins(sdk.NewCoin("alx", sdk.NewInt(200))),
+			acc1.String(): mustParseCoins("1" + stakingDenom),
+			acc2.String(): mustParseCoins("2" + stakingDenom),
+			acc3.String(): mustParseCoins("4blx"),
+			authtypes.NewEmptyModuleAccount(stakingtypes.BondedPoolName).Address:    mustParseCoins("100" + stakingDenom),
+			authtypes.NewEmptyModuleAccount(stakingtypes.NotBondedPoolName).Address: mustParseCoins("200" + stakingDenom),
 		},
+		vesting: mustParseCoins("150blx,150" + stakingDenom),
 	}
+
+	// Test that supply has been initialized as expected
+	require.Equal(t, "453", bkMock.GetSupply(ctx).GetTotal().AmountOf(stakingDenom).String())
+	require.Equal(t, "154", bkMock.GetSupply(ctx).GetTotal().AmountOf("blx").String())
+
```

Comment:
A non-mocking test would be nice that validates the spendable supply is correct after a block increases/reduces vesting balances.